### PR TITLE
Fix 64 hardcoded dev

### PR DESCRIPTION
**Pull request by Zenidd**
Hi team,

- This PR allows using Elastic and Wazuh versions differents from the latest ones.
- This PR removes the custom welcome on demo environment.
- This PR removes the "Getting ready the architecture on AWS" blog files. 
- This PR is related to #64.

Greetings,
JP

### CODE REVIEW

File: demo/elastic/wazuh_cf_bootstrap_elastic.sh [hunk 1]
```diff
@@ -18,7 +18,7 @@ master_ip=$(cat /tmp/wazuh_cf_settings | grep '^MasterIp:' | cut -d' ' -f2)
 worker_ip=$(cat /tmp/wazuh_cf_settings | grep '^WorkerIp:' | cut -d' ' -f2)
 kibana_ip=$(cat /tmp/wazuh_cf_settings | grep '^KibanaIp:' | cut -d' ' -f2)
 
-TAG="v3.13.0"
+TAG="v3.13.1"
```
Comment (manuasir) [inline demo/elastic/wazuh_cf_bootstrap_elastic.sh:21]: Why aren't we using this as a parameter?

File: demo/elastic/wazuh_cf_kibana.sh [hunk 2]
```diff
@@ -444,12 +427,11 @@ main(){
   enable_kibana
   optimize_kibana
   start_kibana
-  sleep 60
+  sleep 200
```
Comment (manuasir) [inline demo/elastic/wazuh_cf_kibana.sh:430]: Please explain why this is necessary.

File: demo/elastic/wazuh_cf_bootstrap_elastic.sh (hunk 1 above)
Comment (Zenidd) [reply demo/elastic/wazuh_cf_bootstrap_elastic.sh]: Missed that one. Thanks! Fixed on 19cccdadcbace719c7ce9c3cca7ba49c355230c1

File: demo/elastic/wazuh_cf_kibana.sh (hunk 2 above)
Comment (Zenidd) [reply demo/elastic/wazuh_cf_kibana.sh:430]: During kibana optimize, the `wazuh.yml` file of the Wazuh APP is copied and after finishing the optimize it is mistakenly replaced with a default one. 

This sleep allows the optimize to finish and then the `wazuh.yml` file is copied in time, avoiding being replaced by the default one.

**Pull request merged**